Exercise_A/index.html

```diff
@@ -10,6 +10,7 @@
   </head>
   <body>
     <!-- CODE HERE -->
+    <a href="about.html"> <img src="../image/flower-icon.png" alt="flower"></a>
 
     <script src="" async defer></script>
   </body>
```

Comment:
Hello Chloe, 
Great work with this, the user would be able to click the image and it's linked to the next page!